Polyhedron/demo/Polyhedron/Plugins/Point_set/Point_set_normal_estimation_plugin.cpp

```diff
@@ -24,12 +24,6 @@
 
 #include "ui_Point_set_normal_estimation_plugin.h"
 
-#if BOOST_VERSION == 105700
```

Comment:
I think this is not correct too. @sgiraudot do you remember why we disable this plugin with boost 1.57 and C++11? Is it still valid? Actually @lrineau did bd435457178c23a2363eb9e05df1f8f63175660d. Shall I revert the fix?

Reply:
Four years ago I should I known better and write comments so that even myself can understand the patch a few years later. But did not, and I do not remember the issue.

I suspect it was a runtime error somewhere in Boost-1.57.

Reply:
I decided to ignore the issue, it is in the demo.